### PR TITLE
use whoami instead of status

### PR DESCRIPTION
**Pull request by joshblum**

### CODE REVIEW

File: pykeybasebot/bot.py [hunk 1]
```diff
@@ -131,18 +131,18 @@ def kvstore(self):
 
     async def _is_initialized(self):
         if not self._initialized:
-            res = await self.submit("status --json")
+            res = await self.submit("whoami --json")
```
Comment (xgess) [inline pykeybasebot/bot.py]: doesn't look like `whoami --json` works. maybe we drop the json and read the result right into username?

Comment (joshblum) [reply pykeybasebot/bot.py]: need to update your client

Comment (joshblum) [reply pykeybasebot/bot.py]: https://github.com/keybase/client/pull/22699

Comment (xgess) [reply pykeybasebot/bot.py]: :+1: then this needs to come with a version bump, and we can't release it until the keybase client has actually been released.

**Pull request merged**